app/controllers/course/achievement/condition/achievements_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Achievement::Condition::AchievementsController <
+  Course::Condition::AchievementsController
+  include Course::AchievementConditionalConcern
```

Comment:
I still don't see the point of this. It can be handled in the base AchievementsController.